### PR TITLE
fix(cypress): remove parallel runs

### PR DESCRIPTION
**Pull request by raisedadead**
This PR temporarily removes the parallel runs. I have explained the details to Oliver, and we will be able to re-enable this soon.

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-name: Cypress - Pull Request
-on: [pull_request]
+name: Cypress
+on: [push, pull_request]
```
Comment (nhcarrigan) [inline .github/workflows/cypress.yml:2]:
```suggestion
on: 
  pull_request:
    branches_ignore:
      - 'renovate/**'
  push:
```

Comment (nhcarrigan) [reply .github/workflows/cypress.yml:2]: To exclude `push` events on Renovate's PRs.

File: .github/workflows/cypress.yml [hunk 2]
```diff
@@ -1,5 +1,9 @@
-name: Cypress - Pull Request
-on: [pull_request]
+name: Cypress
+on: 
+  push:
+    branches_ignore:
```
Comment (nhcarrigan) [inline .github/workflows/cypress.yml:4]:
```suggestion
    branches-ignore:
```
If only I could get things right on the first try 😉

**Review by naomi-lgbt — APPROVED** [review]

**Comment by Sembauke** [comment]
@raisedadead So #42138 is almost ready to go, running it unparalleled would not be a good idea.  Can we reenable parallelization?

**Comment by raisedadead** [comment]
@ojeytonwilliams could you please guide Sem based on what we had discussed in the past.

**Comment by ojeytonwilliams** [comment]
Sure. From what I remember the key problem was that the actions both ran too often (because of renovate) and ran too long (because each parallel run builds the client).    With that in mind, I think a reasonable strategy is to only run these specific tests on pushes to `main` and not have too many containers.  @Sembauke is it enough to have 3 containers, in your experience?

That might still be too much, but I think it's worth trying.

**Comment by Sembauke** [comment]
> @Sembauke is it enough to have 3 containers, in your experience?

Yes, 3 containers should do the trick, it would run 800+ challenges in about 20m.

**Comment by Sembauke** [comment]
What about instead of running all Cypress e2e tests for every pull request, create a pipeline that will be triggered once a day? That would make everything faster.

**Comment by ojeytonwilliams** [comment]
> What about instead of running all Cypress e2e tests for every pull request, create a pipeline that will be triggered once a day? That would make everything faster.

As a rule, we want to be able to deploy `main` at any point and that means we want all the tests to have run on it.  We could fallback to scheduled runs, but I think that's a last resort.

**Pull request merged**